Caboodle/Types/PlacemarkExtensions.android.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Caboodle
+{
+	public static partial class PlacemarkExtensions
+	{
+        private static readonly DateTime epoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
This may be useful for other APIs, maybe we could move this to an internal property on `Platform` or another `Utils` type. In fact, the `Utils` type can be used to store members that are internal, but used across APIs - such as date conversion, maybe some "cached" math operations (eg: 2*PI or something)